Assets/Scripts/Models/Events/HeadlineGenerator.cs

```diff
@@ -40,7 +33,7 @@ public void AddHeadline(string headline, bool displayImmediately = true, bool ke
     {
         if (keepInQueue)
         {
-            headlines.Add(headline);
+            PrototypeManager.Headline.Add(new Headline(headline));
```

Comment:
Because we can, doesn't mean we should. I don't think it's the best pattern, but I may be wrong.

For me prototype shouldn't change after load. If we want a variance, we should store the data in another form.